services/core-web/src/components/noticeOfWork/applications/administrative/NOWApplicationAdministrative.js

```diff
@@ -65,8 +65,8 @@ export const NOWApplicationAdministrative = (props) => {
       >
         <NOWDocuments
           documents={props.noticeOfWork.documents.filter(
-            ({ now_application_document_sub_type_code }) =>
-              now_application_document_sub_type_code === "GDO"
+            ({ mine_document, now_application_document_sub_type_code }) =>
+              mine_document.mine_document_guid && now_application_document_sub_type_code === "GDO"
```

Comment:
What is the point of this change and also this one is not null safe when the other one is.